contrib/cmd/recvtty/recvtty.go

```diff
@@ -61,28 +61,35 @@ terminals:
 )
 
 func bail(err error) {
-	fmt.Fprintf(os.Stderr, "[recvtty] fatal error: %v\n", err)
+	_, _ = fmt.Fprintf(os.Stderr, "[recvtty] fatal error: %v\n", err)
```

Comment:
This change is not inline with what the commit message says. Please split your commits carefully.